docs/marketplace/dev/glue-api-guides/202200.0/shopping-lists/managing-shopping-list-items.md

```diff
@@ -0,0 +1,559 @@
+---
+title: Managing shopping list items
+description: Learn how to manage shopping list items via Glue API.
+last_updated: Jun 16, 2021
+template: glue-api-storefront-guide-template
+related:
+  - title: Managing shopping lists
+    link: docs/marketplace/dev/glue-api-guides/page.version/shopping-lists/managing-shopping-lists.html
+---
+
+This endpoint allows managing shopping list items.
+
+## Installation
+
+For detailed information on the modules that provide the API functionality and related installation instructions, see:
+* [Glue API: Shopping Lists feature integration](/docs/scos/dev/feature-integration-guides/{{page.version}}/glue-api/glue-api-shopping-lists-feature-integration.html)
+* [Glue API: Products feature integration](/docs/scos/dev/feature-integration-guides/{{page.version}}/glue-api/glue-api-product-feature-integration.html)
+
+## Add items to a shopping list
+
+To add items to a shopping list, send the request:
+
+***
+`POST` **/shopping-lists/*{% raw %}{{{% endraw %}shopping_list_id{% raw %}}}{% endraw %}*/shopping-list-items**
+***
+
+| PATH PARAMETER | DESCRIPTION |
+| --- | --- |
+| ***{% raw %}{{{% endraw %}shopping_list_id{% raw %}}}{% endraw %}*** | Unique identifier of a shopping list to add items to. |
+
+### Request
+
+| HEADER KEY | TYPE | REQUIRED | DESCRIPTION |
+| --- | --- | --- | --- |
+| Authorization | string | ✓ | String containing digits, letters, and symbols that authorize the company user. [Authenticate as a company user](/docs/scos/dev/glue-api-guides/{{page.version}}/managing-b2b-account/authenticating-as-a-company-user.html#authenticate-as-a-company-user) to get the value.  |
+
+
+| QUERY PARAMETER | DESCRIPTION | POSSIBLE VALUES |
+| --- | --- | --- |
+| include | Adds resource relationships to the request. | concrete-products |
+
+
+<details>
+<summary markdown='span'>Request sample: add items to the shopping list</summary>
+
+`POST http://glue.mysprykershop.com/shopping-lists/ecdb5c3b-8bba-5a97-8e7b-c0a5a8f8a74a/shopping-list-items`
+
+```json
+{
+    "data": {
+        "type": "shopping-list-items",
+        "attributes": {
+            "quantity": 4,
+            "sku": "005_30663301"
+       }
+    }
+}
+```
+</details>
+
+
+<details>
+<summary markdown='span'>Request sample: add items to the shopping list, and include information about the concrete products</summary>
```

Comment:
Since there are too many variations of how the request can be sent, I would suggest avoiding mentioning relations in the `update/delete/patch` requests (`GET` request samples - is the best place for them IMO). Just a basic scenario is enough.

Reply:
will consider that, thanks!